Update device-locate.md (#2506)
MicrosoftDocs/memdocs: memdocs/intune/remote-actions/device-locate.md

- Use a *lost device sound alert* on a phone to help a user locate the device should it be behind a chair or otherwise out-of-sight.
- Report on the time and location of a devices *last known location*, which can help when the device is off-line, or can’t be reached to verify where it is at. *(Android Enterprise dedicated devices only)*

> Windows location services need to enabled during Windows Out of Box Experience(OOBE), by the user, or using policy [Privacy > LetAppsAccessLocation](/windows/client-management/mdm/policy-csp-privacy#privacy-letappsaccesslocation) for them to be able to locate the device.

Reviewer: ```suggestion
You need to enable Windows location services in Windows Out of Box Experience (OOBE) or by using the [Privacy > LetAppsAccessLocation](/windows/client-management/mdm/policy-csp-privacy#privacy-letappsaccesslocation) policy to be able to locate the device.
```
